pkg/segment/segment.go

```diff
@@ -216,3 +218,43 @@ func IsTelemetryEnabled(cfg *preference.PreferenceInfo) bool {
 	}
 	return false
 }
+
+// sanitizeUserInfo sanitizes username from the error string
+func sanitizeUserInfo(errString string) string {
+	user1, err1 := user.Current()
+	if err1 != nil {
+		return errors.Wrapf(err1, err1.Error()).Error()
```

Comment:
I would suggest if we cannot get the user, we just return the error as is?
this function is about santising the userInfo and if we cannot get the user in my opinion we shouldn't add that to the error because lets say there is an error getting the current user you would send an error like "X component not found: cannot resolve user".

Reply:
Right of course. It doesn't really make sense to wrap the error like that either, it isn't really doing anything!

Reply:
Done.

Reply:
Awesome